content/source/docs/enterprise/private/faq.html.md

```diff
@@ -54,41 +53,39 @@ PTFE has the ability to automatically generate a support bundle including logs a
 * [AMI-based instance bundle generation](./diagnostics.html#ami-based-installs)
 * [Installer-based instance bundle generation](./diagnostics.html#installer-based-instances)
 
----
-
 ## Managing Tool Versions
 
 Terraform Enterprise has a control panel that allows admins to manage the versions of Terraform and Packer and their download locations.
 
-This control panel is available at the `/admin/tools` path or as a link in the sidebar from the general administrative interface at `/admin/manager`.
+In older Terraform Enterprise installations (prior to 201807-1) this control panel is available at the `/admin/tools` path or as a link in the sidebar from the general administrative interface at `/admin/manager`.
```

Comment:
should we call out where this in new versions?

Reply:
My intent was for https://github.com/hashicorp/terraform-website/pull/436/files#diff-7d2bea6f8ee60c85d2b32b9e1a1d8ebfR72 to serve the purpose. Does it need to be closer to the top?

Reply:
NOPE that's fine 🙆